### PR TITLE
icub3: comment out the ft in the legs that are not yet mounted in the real robot

### PR DESCRIPTION
**Pull request by Nicogene**
As per title.

I kept the fixed joints, it should be harmless.

Moreover I set the height of the origin of the model to 64 cm also in `iCubGenvoa09`(~ the height of the root link from the ground)

cc @S-Dafarra @pattacini

### CODE REVIEW

**Comment by pattacini** [comment]
cc @prashanthr05

File: simmechanics/data/icub3/ICUB_3_all_options.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 robotName: iCub
 
 # Model base origin
-originXYZ: [0.0,0.0,0.80]
+originXYZ: [0.0,0.0,0.64]
```
Comment (traversaro) [inline simmechanics/data/icub3/ICUB_3_all_options.yaml:5]: Can you add a line on this change in the PR description?

**Comment by traversaro** [comment]
> Moreover I set the height of the origin of the model to 64 cm also in iCubGenvoa09(~ the height of the root link from the ground)

Is that ok for you @GiulioRomualdi @prashanthr05 ?

File: simmechanics/data/icub3/ICUB_3_all_options.yaml (hunk 1 above)
Comment (Nicogene) [reply simmechanics/data/icub3/ICUB_3_all_options.yaml:5]: I uniformed it with the `iCubGazeboV3`, the change on the height was done by @GiulioRomualdi (https://github.com/robotology/icub-models-generator/blame/b4d31861b14ea1d0cbc4e5952091633f708fdcc8/simmechanics/data/icub3/ICUB_3_all_options_gazebo.yaml.in#L5)

**Comment by prashanthr05** [comment]
> > Moreover I set the height of the origin of the model to 64 cm also in iCubGenvoa09(~ the height of the root link from the ground)
> 
> Is that ok for you @GiulioRomualdi @prashanthr05 ?

Ok for me. It did seem like it was falling from really high when dropping the model. So I suppose 0.64 should be fine.

**Pull request merged**